src/recordlinker/schemas/link.py

```diff
@@ -69,23 +110,14 @@ class LinkFhirInput(pydantic.BaseModel):
     )
 
 
-class LinkFhirResponse(pydantic.BaseModel):
+class LinkFhirResponse(LinkResponse):
     """
     The schema for responses from the link FHIR endpoint.
     """
 
-    found_match: bool = pydantic.Field(
-        description="A true value indicates that one or more existing records "
-        "matched with the provided record, and these results have been linked."
-    )
-    updated_bundle: dict = pydantic.Field(
-        description="If link_found is true, returns the FHIR bundle with updated"
-        " references to existing Personresource. If link_found is false, "
+    updated_bundle: dict | None = pydantic.Field(
+        description="If 'prediction' is 'match', returns the FHIR bundle with updated"
+        " references to existing Person resource. If 'prediction' is 'no_match', "
         "returns the FHIR bundle with a reference to a newly created "
```

Comment:
Can we also add that in the case of a 'possible_match', updated_bundle will be `null`.

Reply:
Oh never mind, I see that comment below!